components/insight/SRC/org/openmicroscopy/shoola/util/ui/colourpicker/LookupTableItem.java

```diff
@@ -99,6 +125,46 @@ public boolean matchesFilename(String filename) {
 
     @Override
     public int compareTo(LookupTableItem o) {
-        return this.readableName.compareTo(o.readableName);
+        return this.label.compareTo(o.label);
+    }
+
+    @Override
+    public int hashCode() {
+        final int prime = 31;
```

Comment:
FYI the top of https://github.com/google/guava/wiki/CommonObjectUtilitiesExplained has some nice stuff offering convenience.